Source/TK.CustomMap/TK.CustomMap.csproj

```diff
@@ -23,7 +23,7 @@
     <None Remove="TK.CustomMap.nuget.targets" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
-    <PackageReference Include="Xamarin.Forms" Version="2.5.0.91635" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Xamarin.Forms" Version="3.2.0.729530-pre2" />
```

Comment:
definitely don't want to be depending on a pre-release version here.

Reply:
i am restoring them :D